backstop_data/engine_scripts/puppet/common/open-accordions.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const Page = require('../page-objects/crm-page.js');
+
+module.exports = async (engine, scenario, vp) => {
+  const page = new Page(engine, scenario, vp);
+
+  console.log('Clicking all accordion headers');
+
+  await page.openAllAccordions();
+
+  try {
+    await engine.waitFor('.loading-text', { hidden: true, timeout: 8000 });
+    await engine.waitFor(1000);
```

Comment:
why is there an arbitrary wait for 1 second? wasn't the previous wait enough? If we could avoid these random waits the test would be faster.

Reply:
it's hardcoded because of the animation, i'm afraid